tqdm/_tqdm.py

```diff
@@ -402,8 +402,14 @@ def inner(df, func, *args, **kwargs):
                 # Precompute total iterations
                 total = getattr(df, 'ngroups', None)
                 if total is None:  # not grouped
-                    total = len(df) if isinstance(df, Series) \
-                        else df.size // len(df)
+                    if isinstance(df, Series):
+                        total = len(df)
+                    elif kwargs.get('axis', 0) == 0:
```

Comment:
I agree that we could disallow *args completely. We should just output a clear message that positional arguments are not supported with tqdm_pandas. This would be the best I think.